Shownotes/Episode2

```diff
@@ -0,0 +1,31 @@
+# 2: Scaling A Codeless Open Source Swift Community with Bas Broek
+
+On this episode, Garric Nahapetian, Chris Lattner, and Bas Broek use Bas's experience as the curator of the Swift Weekly Brief as a jumping off point to discuss the unique challenge of maintaininng an open source project where the community does not necessarily commit a lot of code.
+
+## Hosted by
+
+- [Garric Nahapetian](https://twitter.com/garricn)
+- [Chris Lattner](https://twitter.com/clattner_llvm)
+- [Bas Broek](https://twitter.com/BasThomas)
+
+## How to get involved
+
+Want to contribute to the Swift Community Podcast? We’d love your help and support. This show will be run just like an open source project — and you’re more than welcome to contribute. Check out [the show’s GitHub repository](https://github.com/SwiftCommunityPodcast/podcast) for more information.
+
+## Links
+
+- [Swift Weekly Brief](https://swiftweekly.github.io)
+- [Jesse Squires](https://twitter.com/jesse_squires)
+- [iOS Dev Weekly](https://iosdevweekly.com)
+- [Ole Begemann's PR](https://github.com/SwiftCommunityPodcast/podcast/issues/15)
+- [Gorka Ercilla's PR](https://github.com/SwiftCommunityPodcast/podcast/issues/23)
+- [Mark Malström's PR](https://github.com/SwiftCommunityPodcast/podcast/issues/13)
+- [Julio César Fernández](https://github.com/SwiftCommunityPodcast/podcast/issues/21)
+- [Morten Bek Ditlevsen's comment](https://github.com/SwiftCommunityPodcast/podcast/issues/1#issuecomment-455175831)
+- [See Greg Heo's Brief Issues](https://swiftweekly.github.io/authors)
+- [PR #18](https://github.com/SwiftCommunityPodcast/podcast/pull/18)
```

Comment:
Maybe give these pull requests some more context, so it is easier to understand what they are referencing?

Reply:
No push access, so I created #30. Also adds the `.md` suffix :)

Reply:
git-fu? 😄